src/common/RAJAPerfSuite.cpp

```diff
@@ -159,7 +159,9 @@ static const std::string KernelNames [] =
   std::string("Basic_INIT3"),
   std::string("Basic_INIT_VIEW1D"),
   std::string("Basic_INIT_VIEW1D_OFFSET"),
+  #ifndef RUN_KOKKOS
   std::string("Basic_MAT_MAT_SHARED"),
+  #endif
```

Comment:
This list has to match the list of enum values in the header, avoid ifdefing to avoid confusion.